src/Chapter1.hs

```diff
@@ -520,8 +521,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
-
+closestToZero x y = if abs x == min (abs $ negate x) (abs $ negate y) then x else y
```

Comment:
For absolute value, it doesn't matter what sign the number has, so `negate` is redundant in this case 🙂 

Very interesting solution! 💡 
Though it could be a bit simplified. You see that all you need is to compare the absolute value of both `x` and `y`, and based on that return the result as you do in the `then` and `else` clauses 👍🏼

Reply:
originally it was `(abs (0 - x))`, and it became this after applying the fix hinted by the linter
didn't really give it another thought after seeing it passing the unit test :P